javatests/google/registry/rde/testdata/domain_fragment.xml

```diff
@@ -0,0 +1,14 @@
+<rdeDom:domain
+    xmlns:domain="urn:ietf:params:xml:ns:domain-1.0"
+    xmlns:rdeDom="urn:ietf:params:xml:ns:rdeDomain-1.0">
+  <rdeDom:name>example1.example</rdeDom:name>
+  <rdeDom:roid>Dexample1-TEST</rdeDom:roid>
+  <rdeDom:status s="ok"/>
+  <rdeDom:registrant>jd1234</rdeDom:registrant>
+  <rdeDom:contact type="admin">sh8013</rdeDom:contact>
+  <rdeDom:contact type="tech">sh8013</rdeDom:contact>
+  <rdeDom:clID>RegistrarX</rdeDom:clID>
+  <rdeDom:crRr client="jdoe">RegistrarX</rdeDom:crRr>
+  <rdeDom:crDate>1999-04-03T22:00:00.0Z</rdeDom:crDate>
+  <rdeDom:exDate>2015-04-03T22:00:00.0Z</rdeDom:exDate>
+</rdeDom:domain>
```

Comment:
A really cool test to ensure confidence would be to actually run a domain create flow (with clocks suitably mocked), and then assert that the resulting entities are exactly equal to those created by running an RDE import from a suitable XML fragment, with the only difference being the RDE-specific reason on the history entry of the latter.